.github/ISSUE_TEMPLATE/new_exchange.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Add an exchange
+about: Suggest a new exchange for us to list
+title: ""
+labels: "Type: Feature"
+assignees: ""
+---
+
+Please fill out this short form if you'd like an exchange added to ethereum.org. If it's a decentralized exchange (DEX) you'd like to list, please take a look at our [policy for listing dapps](https://ethereum.org/en/contributing/adding-products/).
+
+**What geographical restrictions apply to this exchange?** 
+
+<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of restricted countries. -->
+
+**Which currencies are accepted by this exchange?**
+
+<!-- If you don't know, please get in touch with the exchange. They'll likey have a list of accepted currencies -->
+
+**Do you have any proof that they're an official company**
+
+<!-- This is a safeguard against listing any malicious sites. You could provide a link to offical company registration. -->
```

Comment:
I would tweak wording here to say "You could provide a link to an "About" page from the exchange that provides more information about their legal entity, or a link to an official company registration". 

For many jurisdictions, there won't be a "linkable" document that attests to the corporate entity. But if there is an about page with corporate/legal info, I would expect that is OK for us.